engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetInputTextureFromFBO.java

```diff
@@ -114,20 +112,13 @@ public StateChange getDefaultInstance() {
     }
 
     private int fetchTextureId() {
-
         // TODO: make checks to verify the FBOs has the requested buffer;
-
         if (inputFbo != null) {
-
             switch (textureType) {
-
                 case ColorTexture:
                     return inputFbo.colorBufferTextureId;
 
-                case DepthRenderBuffer:
-                    return inputFbo.depthStencilRboId;
-
-                case DepthTexture:
+                case DepthStencilTexture:
```

Comment:
The enum (and the corresponding retrieving function) are used for a BindTexture StateChange, so it should only support Textures. depthRenderBuffer is NOT a texture, and so logically shouldn't even be in the list. Even if I return the depthRenderBuffer Fbo, you can't just use it like the other returned values (in a glBindTexture(TEXTURE_2D, ...) call). So, for the sake of clear and logical code, I removed it.